admin_manual/configuration_server/hardening.rst

```diff
@@ -10,6 +10,15 @@ This document lists some security hardenings which require manual interaction by
           however in some cases that are considered second level as defense administrators are encouraged to check these
           hardenings manually.
 
+Deployment
+----------
+
+Move data directory outside of the web root
+*******************************************
+It is highly recommended to move the data directory (where ownCloud stores its data) outside of the web root (i.e. outside
+of ``/var/www``), this can be done by modifying the ``datadirectory`` switch in the configuration file. It is possible to
+do this also after an instance has been installed by moving the folder manually.
```

Comment:
> I think this needs some clarification. At least on Windows it seems to be not supported: owncloud/core#14522

We're going to drop Windows Server support - https://github.com/owncloud/core/pull/14540

Though I should add a comment that this is all Apache2 / Linux focused at the start of this documents. (that said we're not going to add anything else as Apache2 / Linux is the only one we officially support)